base-libs/expat/autobuild/beyond

```diff
@@ -0,0 +1,4 @@
+if [[ -f "$SRCDIR"/doc/xmlwf.1 ]]; then
```

Comment:
Why is this conditional?

Reply:
If this can't be figured out right now, please consider adding a "FIXME" comment and mention it in the commit message.